### ISSUE TITLE
Add lives?

### ISSUE THREAD

**Issue by Arssin** [Arssin/JumpingGame#28]
Need to think about it.

**Comment by karol-wolski** [comment]
Yeah, It will be a great option, because as a player I would like to have more chance to go through the tough part of the game and not having to start all over again 😄
